.github/workflows/native-build.yml

```diff
@@ -75,6 +76,7 @@ jobs:
           prerelease: true
           title: Development Build
           files: |
+            ./avaje-jex-jdk/target/avaje-jex-jdk
```

Comment:
this file does not exist. because graalvm is not generating. 

Btw the graalvm native CI build is not really running at [your case](https://github.com/SentryMan/test-microservice-frameworks/actions/runs/17751064888/job/50445931688) on every commit, because it is eating too much CPU I put some extra check [here](https://github.com/SentryMan/test-microservice-frameworks/blob/38b55549e32c26f68803b082edce4022df451e04/.github/workflows/native-build.yml#L44) meaning if you want your build to generate graalvm build in GHA change a java or pom xml, just empty line will work.